ipaullly/sendIT
# [#161858618] Create GET orders by specific user endpoint

#### What does this PR do?
Creates an endpoint that allows a user to retrieve orders they made
#### Description of Task to be completed?
A user can send a GET request which returns the orders that the specified user placed on the web app 
#### How should this be manually tested?
clone repo into a local directory
```git clone --single-branch -b ft-GET-user-orders-161858618 git@github.com:ipaullly/sendIT.git```
cd into the sendIT directory
activate virtual environment
```source venv/bin/activate```
install dependencies
```pip install -r requirements.txt```
in the home directory execute the test runner to check whether all tests pass
```pytest```
power up POSTMAN and send a JSON body with order details including username to the POST parcels endpoint. now send a GET request with the username as the id and expect a response with the order that user has made 
#### What are the relevant pivotal tracker stories?
#30
